eng/Install-DotNetWorkloads.ps1

```diff
@@ -0,0 +1,146 @@
+param(
```

Comment:
Is "dotnet workload restore" not usable for us? I'll admit this is all new to me but surprised this isn't something supported in the box?

Reply:
I agree, it is surprising. https://github.com/dotnet/sdk/issues/22151 is one of several issues filed related to improving the workload experience, if you're curious to learn more about the current state of managing workloads.